<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formUpdateVoiture</class>
 <widget class="QDialog" name="formUpdateVoiture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>19</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Update voiture</string>
   </property>
   <widget class="QPushButton" name="pushButton_modifier">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Matricule</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Modele</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Couleur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Prix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Categorie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Marque</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="input_matricule"/>
       </item>
       <item>
        <widget class="QLineEdit" name="input_modele"/>
       </item>
       <item>
        <widget class="QLineEdit" name="input_couleur"/>
       </item>
       <item>
        <widget class="QLineEdit" name="input_prix"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_categorie"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_marque"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
